dbt/models/qc/schema.yml

```diff
@@ -221,6 +221,244 @@ models:
+            data_type: int
+          - name: ORATE
+            index: K
+            data_type: float
+          - name: OSIZE
+            index: AB
+            data_type: float
+          - name: Override Decr
+            index: Z
+            data_type: float
+          - name: Override Incr
+            index: Y
+            data_type: float
+          - name: OVRPRICE
+            index: O
+            data_type: int
+          - name: PRICE
+            index: N
+            data_type: int
+          - name: SF
+            index: L
+            data_type: int
+          - name: TAXYR
+            index: B
+            data_type: int
+
+  - name: qc.vw_ic_reference_all_non_res_pin_level_data
+    description: '{{ doc("view_vw_ic_reference_all_non_res_pin_level_data") }}'
+    config: *qc_report_ic_reference_tags
+    meta:
+      export_name: AllNonRes_PINLevelData
+      export_template: *qc_report_ic_reference_export_template
+      export_format:
+        columns:
+          - name: ADRNO
+            index: L
+            data_type: int
+          - name: APRBLDG
+            index: J
+            data_type: int
+          - name: APRLAND
+            index: H
+            data_type: int
+          - name: APRTOT
+            index: I
+            data_type: int
+          - name: COMVAL
+            index: F
+            data_type: int
+          - name: DWELVAL
+            index: K
+            data_type: int
+          - name: OBYVAL
+            index: G
+            data_type: int
+          - name: TAXYR
+            index: B
+            data_type: int
+
+  - name: qc.vw_ic_reference_all_non_res_pin_level_prior_year_vals
+    description: '{{ doc("view_vw_ic_reference_all_non_res_pin_level_prior_year_vals") }}'
+    config: *qc_report_ic_reference_tags
+    meta:
+      export_name: AllNonRes_PINLevel.PriorYearVals
+      export_template: *qc_report_ic_reference_export_template
+      export_format:
+        columns:
+          - name: ADRNO
+            index: L
+            data_type: int
+          - name: APRBLDG
+            index: J
+            data_type: int
+          - name: APRLAND
+            index: H
+            data_type: int
+          - name: APRTOT
+            index: I
+            data_type: int
+          - name: COMVAL
+            index: F
+            data_type: int
+          - name: DWELVAL
+            index: K
+            data_type: int
+          - name: OBYVAL
+            index: G
+            data_type: int
+          - name: TAXYR
+            index: B
+            data_type: int
+          - name: VALASM3
+            index: T
+            data_type: int
+
+  - name: qc.vw_ic_reference_all_towns_com_bldg_detail
+    description: '{{ doc("view_vw_ic_reference_all_towns_com_bldg_detail") }}'
+    config: *qc_report_ic_reference_tags
+    meta:
+      export_name: AllTowns_ComBldgDetail
+      export_template: *qc_report_ic_reference_export_template
+      export_format:
```

Comment:
**issue (blocking):** I'm getting a `ValueError` when running the following:

```bash
uv run python scripts/export_models.py --select tag:qc_report_ic_reference --where "taxyr = '2024'"
```

```
ValueError: invalid literal for int() with base 10: '477.1'
```

I'm guessing it has to do with the changed data types for land? Either way, may be a non-issue once the data type issue (above) is sorted.